core/Pos/Crypto/SecretSharing.hs

```diff
@@ -57,59 +55,66 @@ import           Pos.Crypto.Random   (deterministic, runSecureRandom)
 
 -- | This key is used as public key in VSS.
 newtype VssPublicKey = VssPublicKey
-    { getVssPublicKey :: Pvss.PublicKey
-    } deriving (Show, Eq)
+    { getVssPublicKey :: Scrape.PublicKey
+    } deriving (Show, Eq, Binary.Binary)
+
+-- | Note: this is an important instance, don't change it! It dictates the
+-- order of participants when generating a commitment.
+instance Ord VssPublicKey where
+    compare = comparing Binary.encode
```

Comment:
Hm, perhaps it could be done more efficiently if `Ord` was defined for `Scrape.PublicKey` in `pvss-haskell`.

Reply:
Yep, but it's too late, I guess.